pkgs/development/compilers/dotnet/sdk/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv
+, libunwind
+, openssl
+, icu
+, libuuid
+, zlib
+, curl
+, patchelf
+}:
+
+let
+  rpath = stdenv.lib.makeLibraryPath [ stdenv.cc.cc libunwind libuuid icu openssl zlib curl ];
+  version = "2.0.3";
+in
+  stdenv.mkDerivation rec {
+    name = "dotnet-sdk-${version}";
+    src = fetchTarball "https://dotnetcli.azureedge.net/dotnet/Sdk/2.0.3-servicing-007037/dotnet-sdk-2.0.3-servicing-007037-linux-x64.tar.gz";
```

Comment:
This should use some fixed output abstraction, `fetchurl` is probably most appropriate.